algorithms/a3c/main.py

```diff
@@ -40,80 +54,275 @@
                     help='value loss coefficient (default: 0.5)')
 parser.add_argument('--max-grad-norm', type=float, default=50,
                     help='value loss coefficient (default: 50)')
-parser.add_argument('--seed', type=int, default=1,
-                    help='random seed (default: 1)')
-parser.add_argument('--test-sleep-time', type=int, default=200,
-                    help='number of seconds to wait before testing again (default: 200)')
-parser.add_argument('--num-processes', type=int, default=4,
-                    help='how many training processes to use (default: 1)')
-parser.add_argument('--num-steps', type=int, default=20,
-                    help='number of forward steps in A3C (default: 20)')
-parser.add_argument('--max-episode-length', type=int, default=1000,
-                    help='maximum length of an episode (default: 1000000)')
 parser.add_argument('--no-shared', default=False,
                     help='use an optimizer without shared momentum.')
+parser.add_argument('--num-processes', type=int, default=4,
```

Comment:
Any info on how these processes are distributed within different cores? Or mention/link the library used so people know how it is handled

Reply:
Couldn't we have 1 parameter instead of 2, i.e. num-processes and synchronous? From what I read here I assume that synchronous just means num-processes == 1. Or is it sth else as well?

Reply:
You can have 1 process and asynchronous because it makes a new a thread and you can't debug within it then